docs/guides/blocks/formulas.md

```diff
@@ -11,7 +11,9 @@ Formulas are one of the most basic building blocks in Coda; used to calculate va
 
 ## Structure of a formula
 
-TODO
+A formula definition consists of a set of key-value pairs, which specify the various settings for the formula. Most of these settings are metadata, such as the name, description, parameters and result type. The actual code that is run each time the formula recalculated is specified using in the `execute` key.
+
+<img src="../../../images/formula_structure.png" srcset="../../../images/formula_structure_2x.png 2x" class="screenshot" alt="Mapping of formula coda to formula editor.">
```

Comment:
GH won't let me comment on line 21 but existing code there has an extra "the" before "letters".

Reply:
Good catch, fixed.